tasks/gulp/copy-to-destination.js

```diff
@@ -62,3 +51,70 @@ gulp.task('copy-files', () => {
     .pipe(yamlFiles.restore)
     .pipe(gulp.dest(taskArguments.destination + '/govuk/'))
 })
+
+function generateFixtures (file) {
+  const json = convertYamlToJson(file)
+  const componentName = path.dirname(file.path).split(path.sep).slice(-1).toString()
+  const componentTemplatePath = path.join(configPaths.components, componentName, 'template.njk')
+
+  if (json) {
+    const examplesJson = json.examples
+    const paramsJson = json.params
+
+    if (examplesJson && paramsJson) {
```

Comment:
Minor – does it matter if `json.params` is not truthy? We don't need it to create the fixtures file, and we don't use `paramsJson` anywhere.